cyprecice/cyprecice.pyx

```diff
@@ -484,29 +523,27 @@ cdef class Interface:
         >>> vertices = np.array([[1, 2], [1, 3], [1, 4], [2, 3], [2, 4], [3, 4]])
         >>> vertices.shape
         (6, 2)
-        >>> interface.set_mesh_edges(mesh_name, vertices)
+        >>> participant.set_mesh_edges(mesh_name, vertices)
         """
         check_array_like(vertices, "vertices", "set_mesh_edges")
 
         if not isinstance(vertices, np.ndarray):
             vertices = np.asarray(vertices)
 
         if len(vertices) > 0:
-            size, n = vertices.shape
+            _, n = vertices.shape
             assert n == 2, "Provided vertices are not of a [N x 2] format, but instead of a [N x {}]".format(n)
         elif len(vertices) == 0:
-            size = vertices.shape[0]
-            dimensions = self.get_dimensions()
+            dimensions = self.get_mesh_dimensions(mesh_name)
 
-        cdef np.ndarray[double, ndim=1] _vertices = np.ascontiguousarray(vertices.flatten(), dtype=np.int)
+        cdef vector[int] cpp_vertices = vertices.flatten()
 
-        cdef bytes mesh_name_py_bytes = mesh_name.encode()
+        self.thisptr.setMeshEdges (convert(mesh_name), cpp_vertices)
 
-        self.thisptr.setMeshEdges (<const char*> mesh_name_py_bytes, size, <const int*>_vertices.data)
 
     def set_mesh_triangle (self, mesh_name, first_vertex_id, second_vertex_id, third_vertex_id):
         """
-        Sets mesh triangle from edge IDs
+        Set a mesh triangle from edge IDs
```

Comment:
Can you try to move these minor documentation updates outside of this PR to improve readability of the diff? Not critical, but this PR is already hard enough to read.